apps/curbside_concerts_web/lib/curbside_concerts_web/templates/request/edit.html.eex

```diff
@@ -8,7 +8,10 @@
   <%= hidden_input f, :id %>
   <%= unless f.errors == [], do: inspect(f.errors) %>
 
-    <label class="field">Nominee Address: <%= text_input f, :nominee_address %></label>
+    <label class="field">Nominee Street Address: <%= text_input f, :nominee_street_address %></label>
```

Comment:
Could we at least display-only the old address field above here for maybe another week?

Reply:
I added this in a commit I just added.